<launch>
<!-- Launch file for Livox LiDAR -->

	<arg name="rviz" default="true" />

	<rosparam command="load" file="$(find lio)/config/mid70.yaml" />

	<!-- pkg：使用的包，type：可执行文件的名称，name：节点名称，output：将信息输出到屏幕-->
	<node pkg="lio" type="lio_node" name="laserMapping" output="screen" /> 

	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d /home/neal/projects/lio_ros/src/lio/rviz_cfg/loam_livox.rviz" />
	</group>

</launch>
